webkit_server.py

```diff
@@ -513,9 +513,12 @@ def issue_command(self, cmd, *args):
     self._writeline(cmd)
     self._writeline(str(len(args)))
     for arg in args:
-      arg = str(arg)
```

Comment:
This function cannot handle unicode strings. Try the following:

`str(u"canción")`